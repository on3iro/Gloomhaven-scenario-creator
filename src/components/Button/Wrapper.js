/**
 * Styled container for Button
 *
 * @namespace Button.Wrapper
 * @memberOf Button
  */

import styled from 'styled-components';


const Wrapper = styled.div`
  display: inline-block;
`;

export default Wrapper;
