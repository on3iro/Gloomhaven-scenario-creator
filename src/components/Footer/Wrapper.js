/**
 * Styled container for Footer
 *
 * @namespace Footer.Wrapper
 * @memberOf Footer
  */

import styled from 'styled-components';


const Wrapper = styled.div`
  clear: both;
  min-height: 100px;
  width: 100%;
  margin-top: auto;
  flex: none;
  flex-shrink: 0;
  padding: 30px;
  background-color: #e3e3e3;
`;

export default Wrapper;
