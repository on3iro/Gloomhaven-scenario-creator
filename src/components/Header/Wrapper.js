/**
  * Styled container for Header
  *
  * @namespace Header.Wrapper
  * @memberOf Header
  */

import styled from 'styled-components';


const Wrapper = styled.div`
  margin-bottom: 10px;
  height: 40px;
  width: 100%;
  flex: none;
  background-color: #333;
  color: #fff;
`;

export default Wrapper;
