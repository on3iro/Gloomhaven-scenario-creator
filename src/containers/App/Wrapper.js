/**
  * Styled Wrapper for the whole application
  *
  * @namespace App.Wrapper
  * @memberOf App
  */

import styled from 'styled-components';


const Wrapper = styled.div`
  height: 100%;
  min-height: 100%;
  display: flex;
  flex-direction: column;
`;

export default Wrapper;
