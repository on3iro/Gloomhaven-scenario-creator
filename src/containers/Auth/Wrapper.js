/**
  * Renders a styled div for the HomePage
  *
  * @namespace HomePage.Wrapper
  * @memberOf HomePage
  */

import styled from 'styled-components';


const Wrapper = styled.div`
  margin: 0 auto;
  width: 50%;
`;

export default Wrapper;
