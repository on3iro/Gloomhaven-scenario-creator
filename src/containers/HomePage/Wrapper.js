import styled from 'styled-components';


const Wrapper = styled.section`
  width: 50%;
  margin: 0 auto;
`;

export default Wrapper;
