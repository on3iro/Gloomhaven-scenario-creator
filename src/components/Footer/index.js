/**
 * Footer that renders some content
 *
 * @namespace Footer
  */

import React from 'react';

import { Link } from 'react-router';
import Wrapper from './Wrapper';


const Footer = () => {
  return (
    <Wrapper>
      <Link to="/impressum">Impressum</Link>
      <Link to="/agb">AGB</Link>
    </Wrapper>
  );
};

export default Footer;
