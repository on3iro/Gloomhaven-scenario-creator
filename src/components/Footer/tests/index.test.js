import React from 'react';
import { Link } from 'react-router';
import { shallow } from 'enzyme';

import Footer from '../index';
import Wrapper from '../Wrapper';


describe('<Footer />', () => {
  it('Should render <Wrapper />', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper.find(Wrapper).length).toEqual(1);
  });

  it('Should render react-router Link', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper.find(Link).exists()).toEqual(true);
  });
});
