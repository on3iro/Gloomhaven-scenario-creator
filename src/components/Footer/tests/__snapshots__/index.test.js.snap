exports[`<Footer /> Should render <Wrapper /> 1`] = `1`;

exports[`<Footer /> Should render react-router Link 1`] = `true`;
